daemon/execdriver/lxc/driver.go

```diff
@@ -214,10 +256,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	c.ProcessConfig.Args = append([]string{name}, arg...)
 
 	if err := createDeviceNodes(c.Rootfs, c.AutoCreatedDevices); err != nil {
+		killNetNsProc(proc)
```

Comment:
@crosbymichael I know, but in this function we wait on the waitLock right here:

```
        <-waitLock
        exitCode := getExitCode(c)
```

So you cannot put the `killNetNsProc` inside a defer